src/components/ReadMore/ReadMore.tsx

```diff
@@ -31,16 +33,54 @@ export const ReadMore: React.FC<ReadMoreProps> = (
 ): JSX.Element => {
   const { textHidden, linkButton } = useStyles();
   const _isHidden = props.isHidden ?? true;
-  const [isHidden, setIsHidden] = useState(_isHidden);
+  const [openTooltip, setOpenTooltip] = useState(false);
+  const { isMobile } = useMobileDetect();
+
+  const handleTooltipClose = () => {
+    setOpenTooltip(false);
+  };
+
+  const handleTooltipOpen = () => {
+    setOpenTooltip(true);
+  };
 
   return (
-    <div>
-      <div className={isHidden ? textHidden : undefined}>{props.children}</div>
+    <span>
+      <div className={_isHidden ? textHidden : undefined}>{props.children}</div>
       {_isHidden && (
-        <button className={linkButton} onClick={() => setIsHidden(!isHidden)}>
-          {!isHidden ? 'see less' : 'see more'}
-        </button>
+        <>
+          {isMobile ? (
```

Comment:
I'm not sure if this is the best way to do this - it might be - but it does feel a little brittle to me... what are the best practices for mobile design around this?

Reply:
I'm pretty sure that this is a good practice for this particular case, I don't see another way because I'm switching to use 2 different way to implement the tooltip as explain above, in mobile is not posible onhover event to show the tooltip at least we can just go for the click event to show it.

Noticed in this is not just detect the sizes of the screen is to detect if is mobile because I need to know if the device does has touch feature, for just size we can use the actual hooks from material-ui

example:

`import { useTheme } from '@mui/material/styles';
import useMediaQuery from '@mui/material/useMediaQuery';

function MyComponent() {
  const theme = useTheme();
  const matches = useMediaQuery(theme.breakpoints.down('sm'));

  return <span>{`theme.breakpoints.down('sm') matches: ${matches}`}</span>;
}`

I'm using this for something that I started to work about the top navbar menu on mobile

Reply:
@f1lander since this is basically a pattern for a tooltip that you hover on desktop and tap on mobile, do you want to move this behavior into ApeInfoTooltip itself? We would want all tooltips to have this behavior, no?

Reply:
@levity you right, that would be better! I'll do it.